src-plugins/medClut/medClutToolBox.cpp

```diff
@@ -116,84 +115,70 @@ medClutToolBox::~medClutToolBox(void)
 
 void medClutToolBox::updateView()
 {
-    medTabbedViewContainers * containers = this->getWorkspace()->stackedViewContainers();
-    QList<medViewContainer*> containersInTabSelected = containers->containersInTab(containers->currentIndex());
-    medAbstractView *view=NULL;
-    for(int i=0;i<containersInTabSelected.length();i++)
-    {
-        if (containersInTabSelected[i]->isSelected())
-        {
-            view = containersInTabSelected[i]->view();
-            break;
-        }
-    }
+    medAbstractView *view = this->getWorkspace()->stackedViewContainers()->getFirstSelectedContainerView();
 
-    if(!view)
+    if(view)
     {
-        return;
-    }
-
-    medAbstractLayeredView * medView = dynamic_cast<medAbstractLayeredView *> (view);
-    if ( !medView )
```

Comment:
i just removed this test, since we test `view` just before. This paragraph does not change (except the indentation) :)